### ISSUE TITLE
complex-styles Broken with latest Upgrade

### ISSUE THREAD

**Issue by pietromarrone** [atmist/snazzy-info-window#41]
After latest upgrade to: 
- @agm/core@1.0.0-beta.3
- @agm/snazzy-info-window@1.0.0-beta.3

The example: [complex-styles](https://rawgit.com/atmist/snazzy-info-window/master/examples/complex-styles/index.html)
the width is not coherent.

**Comment by joshcomley** [comment]
Yes, the widths are not being set dynamically any more on any examples (or in my own attempts)
